beetmoverscript/script.py

```diff
@@ -86,6 +95,46 @@
     await retry_download(context=context, url=source, path=abs_file_path)
     await retry_upload(context=context, destinations=destinations, path=abs_file_path)
 
+    if update_balrog_manifest:
+        context.balrog_manifest.append(
+            enrich_balrog_manifest(context, abs_file_path, locale, destinations)
+        )
+
+
+def enrich_balrog_manifest(context, path, locale, destinations):
+    props = context.properties
+
+    if props["branch"] == 'date':
```

Comment:
I'd suggest we move all the hardcoded values like `date` in constants file to ease any potential tweaking later on if we ever change the dev branch. Or even add another one. But I think it's totally fine for now.